.circleci/config.yml

```diff
@@ -95,7 +95,8 @@ commands:
           no_output_timeout: 16m
           command: |
             mkdir -p ${SCREENSHOT_DIR}
-            yarn test --ci --maxWorkers=2 --reporters=default --reporters=jest-junit
+            TESTS_TO_RUN=$(circleci tests glob "jest/**.integration-test.js" | circleci tests split --split-by=timings)
```

Comment:
What is this `circleci` command and what does it do?

Reply:
See: https://circleci.com/blog/a-guide-to-test-splitting/

Reply:
(and further https://circleci.com/docs/2.0/parallelism-faster-jobs/)

Reply:
Basically, we tell it all of the test files and have it break them up into the number of groups specified by the `parallelism` key below, using historical timing data from previous test results as the basis for the split.